src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,47 @@
+import Button from "../../components/Button/Button";
+import "./Profile.scss";
+import { Link } from "react-router-dom";
+import { UserProfile } from "./mockData";
+
+type ProfileProps = {
+  user: UserProfile;
+};
+
+const Profile = ({ user }: ProfileProps) => {
+  const starredPassword = "*".repeat(user.password.length);
+  return (
+    <div className="profile">
+      <img src={user.img} className="profile__img" />
+      <h1 className="profile__score">{user.score}</h1>
+      <section className="profile__info">
+        <p>
+          <strong>Name</strong> : {user.name}{" "}
+        </p>
+        <p>
+          <strong>Bio</strong> : {user.bio}{" "}
+        </p>
+        <p>
+          <strong>Email</strong> : {user.email}{" "}
+        </p>
+        <p>
+          <strong>Password</strong> : {starredPassword}{" "}
```

Comment:
The password will never be retrieved from the database so this should be a static value, e.g. "**********"

Reply:
Done